source/test/TestApp/TestApp.cpp

```diff
@@ -92,7 +92,7 @@ namespace PlayFabUnit
         pfApiTest.SetTitleInfo(testTitleData);
         testRunner.Add(pfApiTest);
 
-#if !defined(PLAYFAB_PLATFORM_GDK) && false // These tests are still too unstable, and despite passing nearly 100% in debug, still fail 100% in release
```

Comment:
>false [](start = 38, length = 5)

nit : this condition will always resolve to false anyways.

Reply:
Yeah, Intellisense was saying it was still being compiled, after I made this fix things ran just fine.